test/unit/registry-domain-options-sanitiser.js

```diff
@@ -1,22 +1,30 @@
 'use strict';
 
 const expect = require('chai').expect;
+const _ = require('lodash');
 
 describe('registry : domain : options-sanitiser', () => {
-
   const sanitise = require('../../src/registry/domain/options-sanitiser');
 
-  describe('when verbosity is undefined', () => {
-
+  describe('when options is empty', () => {
     const options = {};
-
-    it('should set it to 0 as default', () => {
-      expect(sanitise(options).verbosity).to.equal(0);
+    const defaults = {
+      prefix: '/',
+      tempDir: './temp/',
+      hotReloading: false,
+      verbosity: 0,
+      customHeadersToSkipOnWeakVersion: [],
+      timeout: 120000
+    };
+
+    _.each(defaults, (value, property) => {
+      it(`should set ${property} to ${JSON.stringify(value)}`, () => {
+        expect(sanitise(options)[property]).to.eql(value);
+      });
```

Comment:
I wanted to test `timeout` returning the default, but I just ended up testing each property of the snapshot. When migrating to `jest`, this will be so much prettier ❤️